packages/vite/src/node/server/index.ts

```diff
@@ -255,6 +263,11 @@ export interface ViteDevServer {
    * @param forceOptimize - force the optimizer to re-bundle, same as --force cli flag
    */
   restart(forceOptimize?: boolean): Promise<void>
+  /**
+   * Listen to `process.stdin` for pre-defined keyboard shortcuts, which are
+   * printed to the terminal by this method.
```

Comment:
What can be useful for non standard custom usage if separating biding from help display with a little boolean flag on the method